### ISSUE TITLE
[UX] Project Browser: Merge the module/theme/layout **UPDATE** pages into a single primary tab of a single top-level menu item.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1781]
This was originally proposed in https://github.com/backdrop/backdrop-issues/issues/1399#issuecomment-204861385 but moved here as a follow-up issue. It kinda depends on #1780 though so it's more of a follow-up to that one...

For each one of tor modules, themes and layouts pages, we currently have an "Update" tab. Notice anything?:

![backdrop-update_tabs_for_modules_themes_and_layouts-all_the_same](https://cloud.githubusercontent.com/assets/2423362/14230553/8e32bc06-f9a0-11e5-80d8-b2706c1dd59e.gif)

IT'S THE SAME THING!!

So, how about we consolidate all these into a single tab in the "Extend Backdrop" page and we call it "Updates":

![backdrop-pb-single_updates_tab](https://cloud.githubusercontent.com/assets/2423362/14230582/8c259fb8-f9a1-11e5-9782-cc512e03389b.png)

**Comment by Dinornis** [comment]
What happens to these tabs?
`Structure > Layouts > Settings`
`Functionality > Modules > Uninstall`

**Comment by klonos** [comment]
Hello @Dinornis and sorry for the late reply. I am replacing this issue with #2714.

...but to answer the question, secondary pages would go in secondary tabs (which I know @jenlampton hates 😛)

**Comment by klonos** [comment]
...To be honest though, what I would really like to see is that we keep `Structure` > `Layouts` for managing *layouts* and have the "Manage layouts" tab from my mockups be a "Manage layout *templates*" tab instead. This will keep layouts (the things we create in order to manage blocks) under `Structure`, while layout *templates* (the `type = layout` projects that we download) would live under say a new `Extend Backdrop` menu.
